js/utils/lib/trumbowyg.js

```diff
@@ -12,26 +13,28 @@ export const editorOptions = {
     'btnGrp-lists',
     ['horizontalRule'],
   ],
+
 };
 
 export function installRichEditor(attachPoint, options = {}) {
-  const opts = {
-    editorOptions: {
-      ...editorOptions,
-      ...options.editorOptions || {},
-    },
-    ...options,
+  const language = app.localSettings.get('language');
```

Comment:
Since this isn't used anywhere else, can we just set this inline when setting the `options.editorOptions`?

Reply:
This should be run through `getTrumboLangFileNameByCode` to ensure you are mapping the lang code from settings to the trumbo one since they differ on some codes, for example:

```
options.editorOptions = {
  ...defaultEditorOptions,
  lang: getTrumboLangFileNameByCode(app.localSettings.get('language')),
```